docs/git-guide.md

```diff
@@ -879,8 +880,10 @@ You'll see the *Open a pull request* page:
 
 ![images-create-pr]
 
-Provide a **title** and first comment for your pull request. When ready, click
-the green **Create pull request** to submit the pull request.
+Provide a **title** and first comment for your pull request. Prefix your pull
```

Comment:
Small detail: saying "Prefix..." here again feels a bit redundant to me. What do you think about "Remember to prefix..."?